app/main.py

```diff
@@ -1,3 +1,83 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+import math
+from datetime import datetime
+from typing import Union
+
+from app.car import Car
+from app.customer import Customer
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+    with open("app/config.json", "r") as file:
+        config_file = json.load(file)
+
+    fuel_price = config_file["FUEL_PRICE"]
+    customers_list = [
```

Comment:
Rename customers_list -> customers
suffix _list is redundant
Why you use it?

Reply:
sorry, was used for easier to find out what is the var type inside